app/frontend/src/components/Navigation/ExternalNavButton.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography, TypographyProps } from "@material-ui/core";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  link: {
+    color: theme.palette.text.secondary,
+    display: "flex",
+    flex: "1",
+    fontSize: "2rem",
+    maxWidth: theme.spacing(21),
```

Comment:
```suggestion
    maxWidth: "10.5rem",
```
We've agreed recently that we probably shouldn't be using this `spacing` function for anything that aren't spacing related CSS (padding and margins).